services/node/node-version-color.spec.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const { expect } = require('chai')
+const {
+  getStableVersions,
+  getCurrentVersion,
+  versionColorForRangeLts,
+  versionColorForRangeCurrent,
+} = require('./node-version-color')
+
+describe('node version color', function() {
+  describe(`stable`, function() {
+    it('should only return lts versions', async function() {
+      this.timeout(4000)
```

Comment:
I'd really like to avoid running tests like this that are making live API calls within our unit test suite. It'd probably better to test the logic either by setting up [service tests using nock to mock the API responses](https://github.com/badges/shields/blob/master/doc/service-tests.md#5-mocking-responses) or perhaps keeping these as unit tests but using Sinon to mock out the relevant function calls

Reply:
Will do